ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/CCPA/SPCCPAManagePreferenceViewController.swift

```diff
@@ -112,7 +121,9 @@ extension SPCCPAManagePreferenceViewController: UITableViewDataSource, UITableVi
         guard let sectionComponent = sections[section].header else { return nil }
 
         let label = UILabel(frame: CGRect(x: 0, y: 0, width: tableView.frame.width, height: 50))
-        loadLabelText(forComponent: sectionComponent, label: label)
+        label.numberOfLines = 0
```

Comment:
should `numberOfLines` really be `0`?

Reply:
@andresilveirah yes, it enables multiline for the text